.github/workflows/rspec.yml

```diff
@@ -0,0 +1,28 @@
+name: RSpec
+
+on:
+  push:
+    branches:
+    - "main"
+  pull_request:
+    branches:
+      - main
```

Comment:
For consistency, would you mind fixing the indentation for the branch listing as well as omitting the quotes?